src/openrct2-ui/windows/News.cpp

```diff
@@ -199,37 +199,39 @@ static void window_news_scrollgetsize(rct_window* w, int32_t scrollIndex, int32_
  *
  *  rct2: 0x0066EA5C
  */
-static void window_news_scrollmousedown(rct_window* w, int32_t scrollIndex, ScreenCoordsXY screenCoords)
+static void window_news_scrollmousedown(rct_window* w, int32_t scrollIndex, const ScreenCoordsXY& screenCoords)
 {
     int32_t itemHeight = window_news_get_item_height();
     int32_t i, buttonIndex;
 
     buttonIndex = 0;
+    auto mutableScreenCoords = screenCoords;
     for (i = 11; i < 61; i++)
```

Comment:
While we're here, could you use a constant or expression instead of the hardcoded 61 here?

Reply:
Does it exist already? I'm unsure of what it means.